### PR TITLE
Version 1.5

### PR DESCRIPTION
**Pull request by CharlieSYH**
## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/guides/ingest-from-mysql-cdc.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Data is in Debezium JSON or Debezium AVRO format. [Debezium](https://debezium.io
 
 RisingWave supports creating multiple CDC tables that share a single MySQL CDC source. 
 
-Connect to the upstream database by creating a CDC source using the [`CREATE SOURCE`](/sql/commands/create-source.md) command and MySQL CDC parameters. The data format is fixed as `FORMAT PLAIN ENCODE JSON` so it does not need to be specified.
+Connect to the upstream database by creating a CDC source using the [`CREATE SOURCE`](/sql/commands/sql-create-source.md) command and MySQL CDC parameters. The data format is fixed as `FORMAT PLAIN ENCODE JSON` so it does not need to be specified.
```
Comment (hengm3467) [inline docs/guides/ingest-from-mysql-cdc.md:194]:
```suggestion
Connect to the upstream database by creating a CDC source using the [`CREATE TABLE`](/sql/commands/sql-create-table.md) command and MySQL CDC parameters. The data format is fixed as `FORMAT PLAIN ENCODE JSON` so it does not need to be specified.
```

File: versioned_docs/version-1.4/ingest/ingest-from-kafka.md [hunk 2]
```diff
@@ -377,7 +377,7 @@ To create a Kafka source with a VPC connection, in the WITH section of your `CRE
 |Parameter| Notes|
 |---|---|
 |`privatelink.targets`| The PrivateLink targets that correspond to the Kafka brokers. The targets should be in JSON format. Note that each target listed corresponds to each broker specified in the `properties.bootstrap.server` field. If the order is incorrect, there will be connectivity issues. |
-|`privatelink.endpoint`|The DNS name of the VPC endpoint. <br/> If you're using RisingWave Cloud, you can find the auto-generated endpoint after you created a connection. See details in [Create a VPC connection](/cloud/vpc-create-a-connection.md#whats-next).|
+|`privatelink.endpoint`|The DNS name of the VPC endpoint. <br/> If you're using RisingWave Cloud, you can find the auto-generated endpoint after you created a connection. See details in [Create a VPC connection](cloud/vpc-create-a-connection.md#whats-next).|
```
Comment (hengm3467) [inline versioned_docs/version-1.4/ingest/ingest-from-kafka.md]: why it's "cloud", not "/cloud"? I think the 1.5 version has "/cloud".

**Review by hengm3467 — APPROVED** [review]
Added a comment. Otherwise looks good to me. Thanks! @CharlieSYH

File: versioned_docs/version-1.4/ingest/ingest-from-kafka.md (hunk 2 above)
Comment (CharlieSYH) [reply versioned_docs/version-1.4/ingest/ingest-from-kafka.md]: Gee, thanks!

**Pull request merged**